polkadot/runtime/parachains/src/scheduler.rs

```diff
@@ -292,10 +292,8 @@ impl<T: Config> Pallet<T> {
 				.into_iter()
 				.filter(|(freed_index, _)| (freed_index.0 as usize) < c_len)
 				.for_each(|(freed_index, freed_reason)| {
-					match sp_std::mem::replace(
```

Comment:
What is wrong with sp_std::mem::replace?

Reply:
Maybe some quirk of my local setup, but the sp_std version was producing errors.

Reply:
sp_std is compiling fine after `rustup update`